workers/tests/unittests/test_cli.py

```diff
@@ -57,6 +57,9 @@ def test_create_all_indicators_invlid_opts(self, mo):
         assert result.exit_code == 2
 
     @mock.patch("ohsome_quality_analyst.oqt.create_indicator", mock.AsyncMock())
+    @mock.patch(
+        "ohsome_quality_analyst.geodatabase.client.get_feature_ids", mock.AsyncMock()
+    )
```

Comment:
What was changed to make this mock patch necessary?

Reply:
Good catch. Nothing changed to make it necessary. I removed this mock.